### PR TITLE
Style StackStorm invite form

### PR DESCRIPTION
**Pull request by dzimine**
Here is a form to replace the current TypeForm one.

Suggestions are preferred in form of CSS/HTML to help my struggle with them :)

TODO:
- [x] Base layout/styles in place
- [x] Re-add and style "OK" and "Error" messages

image

image

image

### CODE REVIEW

File: web/index.html [hunk 1]
```diff
@@ -13,27 +13,27 @@
...
+      </div>
+      <div class="tc tl-ns ph3">
+        <h1 class="near-white f1">
+          <span>STACKSTORM'S&nbsp;</span>
+          <span><font color="#ff9212">OPEN SOURCE <br />COMMUNITY</font>
+          RUNS ON SLACK</span>
+        </h1>
+        <h2 class="f5 tc f3-l fw1 mb4 near-white lh-title">Welcome to StackStorm! </h2>
+        <p class="f5 tc f3-l fw1 near-white lh-title">
+        Please introduce yourself, and leave an email <br /> to receive an invite to our Slack community.
```
Comment (LindsayHill) [inline web/index.html:32]: Suggested text:

> Enter your email address below, to receive an invite to our Slack community. Sign up and introduce yourself!

**Comment by Kami** [comment]
The layout is actually not all that bad at all :+1: :)

File: web/index.html (hunk 1 above)
Comment (dzimine) [reply web/index.html:32]: I meant "introduce yourself" right here on the form, as a way to say "enter your name". The "name" field is also desirable. I can make it optional as @alexmakarski suggested, but somehow still care for a name.

**Comment by dzimine** [comment]
Completed, deployed, test it here https://6ktfm1psz9.execute-api.us-west-2.amazonaws.com/dev/index.html

I'll wire it to stackstorm.com/community-signup later today.

**Comment by LindsayHill** [comment]
Couple of things: 

- Page title is "Serverless Slack Invite" - I think it should be "StackStorm Slack Invite"
- Page displayed on form submit does not replace existing page when using Safari on macOS. Instead I need to scroll to the right to see the new content. This is not obvious, especially since scroll bars are deliberately 'hidden' on macOS.

**Comment by dzimine** [comment]
Fixed: 
- [x] Safari Lightbox issue
- [x] Missed last name (client, backend, and added tests)
- [x] Page title
Even managed to wrestle a favicon.ico there (getting it over aws api gateway is fun).

**Comment by dzimine** [comment]
To debug (links for StackStorm folks, obviously): 

1) step-function logs here: https://us-west-2.console.aws.amazon.com/states/home?region=us-west-2#/
2) Lambda dashboard: https://us-west-2.console.aws.amazon.com/lambda/home?region=us-west-2#/ 
3) Lambda function logs (per invocation, sort by most recent): https://us-west-2.console.aws.amazon.com/cloudwatch/home?region=us-west-2#logs:prefix=/aws/lambda/slack-signup

**Pull request merged**